package.json

```diff
@@ -23,7 +24,7 @@
     "repl": "npm run build && node utils/replInit.ts",
     "test:integration:node": "TEST_ENV=node jest src/integration/ -c jest.config.integration-node.js --runInBand --bail",
     "test:integration:edge": "TEST_ENV=edge jest src/integration/ -c jest.config.integration-edge.js --runInBand --bail",
-    "test:integration:cleanup": "npm run build && node utils/cleanupResources.ts",
+    "test:integration:cleanup": "npm run build && npx utils/cleanupResources.ts",
```

Comment:
changed to using `npx` here since with our new setup, we cannot run `.ts` files directly via `node`